packages/create-novu-app/package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "create-novu-app",
+  "version": "0.0.1-alpha.4",
+  "keywords": [
+    "novu"
+  ],
+  "description": "Create Novu-powered apps with one command",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/novuhq/novu"
+  },
+  "author": "Novu Team <engineering@novu.co>",
+  "license": "MIT",
+  "bin": {
+    "create-novu-app": "./dist/index.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "start": "node dist/index.js",
+    "dev": "ncc build ./index.ts -w -o dist/",
+    "prerelease": "node ../../scripts/rm.mjs dist",
+    "release": "ncc build ./index.ts -o ./dist/ --minify --no-cache --no-source-map-register",
+    "build": "pnpm release",
+    "lint-fix": "pnpm prettier -w --plugin prettier-plugin-tailwindcss 'templates/*-tw/{ts,js}/{app,pages}/**/*.{js,ts,tsx}'",
+    "release:alpha": "npm i && npm run build && npm version prerelease --preid=alpha && npm publish"
+  },
+  "devDependencies": {
+    "@types/async-retry": "1.4.2",
+    "@types/ci-info": "2.0.0",
+    "@types/cross-spawn": "6.0.0",
+    "@types/node": "^20.2.5",
+    "@types/prompts": "2.4.2",
+    "@types/tar": "6.1.5",
+    "@types/validate-npm-package-name": "3.0.0",
+    "@vercel/ncc": "0.34.0",
+    "async-retry": "1.3.1",
+    "ci-info": "watson/ci-info#f43f6a1cefff47fb361c88cf4b943fdbcaafe540",
+    "commander": "2.20.0",
+    "conf": "10.2.0",
+    "cross-spawn": "7.0.3",
+    "fast-glob": "3.3.1",
+    "picocolors": "1.0.0",
+    "prettier-plugin-tailwindcss": "0.3.0",
+    "prompts": "2.4.2",
+    "tar": "6.1.15",
+    "update-check": "1.5.4",
+    "validate-npm-package-name": "3.0.0",
+    "typescript": "5.2.2",
+    "async-sema": "3.0.1"
+  },
+  "engines": {
+    "node": ">=18.17.0"
+  }
```

No need for engines here, it's added at the monorepo root

```suggestion
  "engines": {
    "node": ">=18.17.0"
  }
```

---

done